projects/admin/src/app/circulation/patron/card/card.component.html

```diff
@@ -25,7 +25,7 @@
       <div class="col-md-10">
       <h3>
         <a [routerLink]="['/records', 'patrons','detail', patron.pid]">
-          <span *ngIf="patron.last_name">{{ patron.last_name }},</span>
+          <span *ngIf="patron.last_name">{{ patron.last_name }}, </span>
```

Comment:
It could be better to add ", " only if first_name is not empty.

Reply:
Done